clipper_admin/clipper_manager.py

```diff
@@ -441,8 +447,8 @@ def deploy_model(self,
                      version,
                      model_data,
                      container_name,
-                     labels,
                      input_type,
+                     labels=[DEFAULT_LABEL],
```

Comment:
What's the purpose of having all models get a "default" label? Can this be defaulted to `[]`?